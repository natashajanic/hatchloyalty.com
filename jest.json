{
  "collectCoverage": true,
  "coveragePathIgnorePatterns": ["tests"],
  "globals": {},
  "moduleFileExtensions": [
    "ts",
    "tsx",
    "js"
  ],
  "moduleNameMapper": {
    "^.+\\.(css|less|scss)$": "identity-obj-proxy",
    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tests/assetsTransformer.ts",
    "^src/(.*)$": "<rootDir>/src/$1"
  },
  "setupFiles": [
    "raf/polyfill"
  ],
  "setupFilesAfterEnv": [
    "<rootDir>/tests/setupTests.ts"
  ],
  "snapshotSerializers": [
    "enzyme-to-json/serializer"
  ],
  "transform": {
    "\\.(ts|tsx)$": "ts-jest"
  },
  "testRegex": "/src/.*/__tests__/.*\\.(ts|tsx|js)$",
  "unmockedModulePathPatterns": [],
  "verbose": true
}
