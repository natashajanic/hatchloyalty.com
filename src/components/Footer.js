import React from 'react'
import system from 'system-components'

const UIFooter = system({
  is: 'footer',
  alignItems: 'center',
  bg: 'white',
  display: 'flex',
  justifyContent: 'space-between',
  mx: 'auto',
  px: 3,
  py: 3,
  width: '1024px',
})


const Footer = () => (
  <UIFooter>
    Footer
  </UIFooter>
)


export default Footer
