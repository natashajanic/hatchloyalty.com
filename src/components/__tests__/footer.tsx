import { shallow, ShallowWrapper } from 'enzyme'
import 'jest' // tslint:disable-line no-import-side-effect
import * as React from 'react'
import Footer from '../footer'

describe('Footer', () => {
  let mountedComponent: ShallowWrapper<any, any, Footer>

  beforeEach(() => {
    mountedComponent = shallow(<Footer />)
  })

  describe('render', () => {
    it('matches the expected snapshot', async () => {
      expect(mountedComponent).toMatchSnapshot()
    })
  })
})
