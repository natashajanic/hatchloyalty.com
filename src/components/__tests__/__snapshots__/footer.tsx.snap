// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render matches the expected snapshot 1`] = `
<Styled(Component)
  alignItems="center"
  bg="white"
  blacklist={
    Array [
      "css",
      "alignItems",
      "color",
      "bg",
      "display",
      "justifyContent",
      "m",
      "mt",
      "mr",
      "mb",
      "ml",
      "mx",
      "my",
      "p",
      "pt",
      "pr",
      "pb",
      "pl",
      "px",
      "py",
      "width",
    ]
  }
  display="flex"
  is="footer"
  justifyContent="space-between"
  mx="auto"
  px={3}
  py={3}
  width="1024px"
>
  Footer
</Styled(Component)>
`;
